lib/reporter/markdown.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var _ = require('lodash');
+
+var util = require('./util');
+
+function printModulesMarkdown(logger, title, modules) {
+  if (modules.length > 0) {
+    logger('### ' + title + ' Modules');
+    _.each(modules, function (module) {
+      logger('* ' + module.name);
+      logger('  - version ' + module.version);
+      if (module.error) {
+        logger('  * ' + module.error.message);
+      }
+    });
+    logger('');
+  }
+}
+
+function markdown(logger, modules) {
+  if (!(modules instanceof Array)) {
+    modules = [modules];
+  }
+
+  var passed = util.getPassing(modules);
+  var flaky = util.getFlakyFails(modules);
+  var failed = util.getFails(modules);
+
+  if (failed.length === 0) {
+    logger('## 🎉🎉 CITGM Passed 🎉🎉');
```

Comment:
Fun emoji for the markdown reporter. This was [discussed when they were originally merged](https://github.com/nodejs/citgm/pull/40)

More than happy to discuss again if you think they could cause an issue